uiowa/uiowa
# Commencement event importer

Resolves #7566 

# How to test

```
ddev blt ds --site=commencement.uiowa.edu && ddev drush @commencement.local uli node/add/venue
```
- Create a venue or two, matching the title and address info to the location_name and geo data from https://content.uiowa.edu/api/v1/views/events_api.json?display_id=events&filters[enddate][value][date]=01-01-2100&filters[types]=355&filters[department]=7266&items_per_page=100. 
```
ddev drush @commencement.local commencement-events
```
- Check one of the newly created events, making sure the pulled data matches the source and is not editable. Make sure to check an event using the venue you added to see they are linked.
